src/managers/run-strategies/reuse-last-initialized.js

```diff
@@ -1,17 +1,38 @@
+/**
+ * The `reuse-last-initialized` strategy looks for the most recent run that matches particular critiera; if it cannot find one, it creates a new run and immediately executes a set of "initialization" operations. 
+ *
+ * This strategy is useful if you have a custom initialization function in your model, and want to make sure it's always executed for new runs. This strategy is also useful if you have a time-based model and always want the run you're operating on to start at a particular step. For example:
```

Comment:
I'd switch the order of these 2 cases; i.e. "if you have a time-base model" and then "also useful for custom init etc"